### PR TITLE
OP: update module relnotes

### PR DESCRIPTION
**Pull request by dwdougherty**
N.B. I didn't spend much time, if any, copy-editing content. I don't think it's necessary for release notes.

### CODE REVIEW

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisbloom/redisbloom-2.8-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+linkTitle: v2.8 (July 2024)
+min-version-db: '7.4'
+min-version-rs: 7.6 (TBD)
+toc: 'true'
+weight: 94
+---
+
+## Requirements
+
+RedisBloom v2.6.13 requires:
```
Comment (dwdougherty) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisbloom/redisbloom-2.8-release-notes.md:18]:
```suggestion
RedisBloom v2.8.2 requires:
```

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+weight: 90
+---
+## Requirements
+
+RediSearch v2.10.5 requires:
+
+- Minimum Redis compatibility version (database): 7.4
+- Minimum Redis Enterprise Software version (cluster): 7.6 (TBD)
+
+## v2.10 GA (v2.10.5)
```
Comment (dwdougherty) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md:21]:
```suggestion
## v2.10 GA (v2.10.5) (July 2024)
```

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.8-release-notes.md [hunk 3]
```diff
@@ -13,11 +13,28 @@ weight: 91
 ---
 ## Requirements
 
-RediSearch v2.8.13 requires:
+RediSearch v2.8.19 requires:
```
Comment (dwdougherty) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.8-release-notes.md:16]:
```suggestion
RediSearch v2.8.14 requires:
```

**Review by dwdougherty — COMMENTED** [review]
Needs a few fixes.

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+Features:
+
+- Enhancing exact matching queries with `TAG` avoiding escaping special meaning characters using the [simpler syntax](https://redis.io/docs/latest/develop/interact/search-and-query/query/exact-match/#tag-field) `'@tag:{"my-query%term"}'` and `NUMERIC` queries:
+  - [#4802](https://github.com/RediSearch/RediSearch/pull/4802) - Using double quotes to wrap you exact matching query terms such as `@email:{"test@redis.com"}` in `DIALECT 2` ( MOD-7299)
```
Comment (andy-stark-redis) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md:36]:
```suggestion
  - [#4802](https://github.com/RediSearch/RediSearch/pull/4802) - Using double quotes to wrap exact matching query terms such as `@email:{"test@redis.com"}` in `DIALECT 2` ( MOD-7299)
```

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  - [#4676](https://github.com/RediSearch/RediSearch/pull/4676), [#4433](https://github.com/RediSearch/RediSearch/pull/4433) - Enhancing query parser to avoid unnecessary escaping (MOD-5756)
+  - [#4527](https://github.com/RediSearch/RediSearch/pull/4527) - Enhancing exact matching queries for `NUMERIC` using single value `FT.SEARCH idx @numeric:[3456]` (MOD-6623)
+  - [#4802](https://github.com/RediSearch/RediSearch/pull/4802) - Enabling support to single operators for `NUMERIC` queries such as equivalence `==`, difference `!=`, greater than `>` and `>=` and less than `<` and `<=` as in `FT.SEARCH idx '@numeric==3456'` (MOD-6749)
```
Comment (andy-stark-redis) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md:39]:
```suggestion
  - [#4802](https://github.com/RediSearch/RediSearch/pull/4802) - Enabling support for single operators in `NUMERIC` queries such as equivalence `==`, difference `!=`, greater than `>` and `>=` and less than `<` and `<=` as in `FT.SEARCH idx '@numeric==3456'` (MOD-6749)
```

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+- Adding new keywords to support indexing empty values using `INDEXEMPTY` and missing values using `INDEXMISSING` per field in the `SCHEMA` while defining the [index](https://redis.io/docs/latest/commands/ft.create/) with `FT.CREATE`
+  - [#4663](https://github.com/RediSearch/RediSearch/pull/4663), [#4721](https://github.com/RediSearch/RediSearch/pull/4721) - Indexing empty strings values `""` for `TAG` and `TEXT` fields (MOD-6540, MOD-7200)
+  - [#4721](https://github.com/RediSearch/RediSearch/pull/4721) - Updating the query parser to support empty values query for `TEXT` as `FT.SEARCH idx '@text_field:""'`or `FT.SEARCH idx '""'` and for `TAG` as in `FT.SEARCH idx '@tag_field:{""}'` (MOD-7200)
```
Comment (andy-stark-redis) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md:43]: Not sure if I'm reading this one correctly, but it looks a bit odd.
```suggestion
  - [#4721](https://github.com/RediSearch/RediSearch/pull/4721) - Updating the query parser to support empty value queries for `TEXT` as `FT.SEARCH idx '@text_field:""'`or `FT.SEARCH idx '""'` and for `TAG` as in `FT.SEARCH idx '@tag_field:{""}'` (MOD-7200)
```

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  - [#4859](https://github.com/RediSearch/RediSearch/pull/4859) - Expose scores to `FT.AGGREGATE` pipeline (MOD-7190)
+
+- [#4227](https://github.com/RediSearch/RediSearch/pull/4227) - Adding support for new operators `INTERSECT` and `DISJOINT` when querying for `GEOSHAPE` [polygons](https://redis.io/docs/latest/develop/interact/search-and-query/query/geo-spatial/) (MOD-6178)
+
+
+Bug fixes (since 2.10.4):
+- [#4854](https://github.com/RediSearch/RediSearch/pull/4854) - Avoid expansion on stemming from numeric values (MOD-7025)
+
+Improvements (since 2.10.4):
+- [#4865](https://github.com/RediSearch/RediSearch/pull/4865) - Add coverage for cleaning garbage entries at indexing missing fields (MOD-7415)
```
Comment (andy-stark-redis) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.10-release-notes.md:60]: Not sure if this is what's intended here?
```suggestion
- [#4865](https://github.com/RediSearch/RediSearch/pull/4865) - Add coverage for cleaning garbage entries when indexing missing fields (MOD-7415)
```

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisjson/redisjson-2.8-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+toc: 'true'
+weight: 95
+---
+
+## Requirements
+
+RedisJSON v2.8.3 requires:
+
+- Minimum Redis compatibility version (database): 7.4
+- Minimum Redis Enterprise Software version (cluster): 7.6 (TBD)
```
Comment (andy-stark-redis) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisjson/redisjson-2.8-release-notes.md:21]: Is the TBD OK for now? Just noticed it here but it's in the other files too.

File: content/operate/oss_and_stack/stack-with-enterprise/release-notes/redistimeseries/redistimeseries-1.12-release-notes.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+  - Two new [module configuration parameters](https://redis.io/docs/data-types/timeseries/configuration/) are introduced: `IGNORE_MAX_TIME_DIFF` and `IGNORE_MAX_VAL_DIFF`.
+  - Two new similar per-key parameters are introduced: `ignoreMaxTimeDiff` and `ignoreMaxValDiff`.
+  - `TS.ADD`, `TS.INCRBY`, and `TS.DECRBY` now have a new optional argument: `[IGNORE ignoreMaxTimeDiff ignoreMaxValDiff]`.
+    When creating a new time series, these two values are used to set the per-key parameters and override the two module configuration parameters. These values are ignored when specified with an existing time series.
+  - `[IGNORE ignoreMaxTimeDiff ignoreMaxValDiff]` is also supported by `TS.ALTER`.
+  - For each call to `TS.ADD`, if the following condition is met: (series is not a compaction), (the series' `DUPLICATE_POLICY` is `LAST`), (`timestamp` ≥ `max_timestamp`), (`timestamp` - `max_timestamp` ≤ `ignoreMaxTimeDiff`), and abs(`value` - `value_at_max_timestamp`) ≤ `ignoreMaxValDiff`)) then this sample is ignored (not added) to the time series. `max_timestamp` is the maximal timestamp in the time series.
```
Comment (andy-stark-redis) [inline content/operate/oss_and_stack/stack-with-enterprise/release-notes/redistimeseries/redistimeseries-1.12-release-notes.md:39]: There might be better ways to format this but it's quite dense and hard to read as it is.
```suggestion
  - For each call to `TS.ADD`, if the following conditions are met:
      - series is not a compaction
      - the series' `DUPLICATE_POLICY` is `LAST`
      - `timestamp` ≥ `max_timestamp`
      - `timestamp` - `max_timestamp` ≤ `ignoreMaxTimeDiff`
      - abs(`value` - `value_at_max_timestamp`) ≤ `ignoreMaxValDiff`))
    then this sample is ignored (not added) to the time series. `max_timestamp` is the maximal timestamp in the time series.
```

**Review by andy-stark-redis — APPROVED** [review]
Few minor suggestions but basically fine.

**Comment by dwdougherty** [comment]
Thanks @andy-stark-redis !

**Pull request merged**